python/axipcie/_AxiGpuAsyncCore.py

```diff
@@ -134,13 +178,21 @@ def __init__(self,
             ))
 
             self.add(pr.RemoteVariable(
-                name         = f'RemoteReadAddress[{i}]',
+                name         = f'RemoteReadAddressL[{i}]',
                 offset       = 0x200 + i*16,
                 bitSize      = 32,
                 mode         = 'RO',
                 pollInterval = 1,
             ))
 
+            self.add(pr.RemoteVariable(
+                name         = f'RemoteReadAddressH[{i}]',
+                offset       = 0x204 + i*16,
+                bitSize      = 32,
+                mode         = 'RO',
+                pollInterval = 1,
+            ))
+
```

Comment:
@slacrherbst Does it make more sense in the SW side to make this a single 64-bit varaible instead of two 32-bit variables?

Reply:
I thought about it, but the driver interface has to split them...... but its just a status so we could do it on the software side.

Reply:
That's a valid reason to split them in rogue varabile as well